src/appFunctions.js

```diff
@@ -22,6 +32,30 @@ export function findPossiblyOwners(owners, usersApp) {
     ));
 }
 
+export function findCardParentByIdChild(idChild, cards) {
+    let card = null;
+    
+    Object.values(cards).find(array => (
+        card = array.find(item => item.children.includes(idChild))
+    ));
+
+    return card;
+}
+
+export function findAvalibleChildsToAdd(id, cards) {
+    const generalArrayCards = Object.values(cards).reduce((prev, item) => [...prev, ...item]);
```

Comment:
why do you need `.reduce` here? `Object.values` will return the same result

Reply:
i didn't know